Testing/Functional/Products/Tests/CallProductTests.ps1

```diff
@@ -0,0 +1,31 @@
+# The purpose of this script is to enable a GitHub Action workflow to
+# run the functional tests for all the products.
+# NOTE: At the moment it only does this for one product, Sharepoint.  Others
+#       will be added.
+# To run this script, call it from the root of the repo, like so:
+# ./Testing/Functional/Products/Tests/CallProductTests.ps1 <params> <thumbprint>
```

Comment:
Use PowerShell built in Comment Based Help so user can Get-Help on file usage: 
Get-Help .\Testing\Functional\Products\Tests\CallProductTests.ps1
 
```suggestion
<#
    .SYNOPSIS
    The purpose of this script is to enable a GitHub Action workflow to run the functional tests for all the products.
    .NOTE: 
    At the moment it only does this for one product, Sharepoint.  Others will be added.
    .EXAMPLE 
    To run this script, call it from the root of the repo, like so:
    ./Testing/Functional/Products/Tests/CallProductTests.ps1 <params> <thumbprint>
#>
```

Reply:
Done.